### ISSUE TITLE
End of line in the readme files is unpredictable, this creates clipping when showing from within the client

### ISSUE THREAD

**Issue by stefson** [Return-To-The-Roots/s25client#1044]
This is proably due to the font used - it seems to be a somewhat proportional, non-mono font. For the lines consisting of `-`, 81 of those will be rendered into one line by the client. 

For all the lines with actuall text in them, it is kind of a gamble to predict wether the line-break will be at the correct spot, or not. 

See the screenshot for an example of this: 

![2019-02-21-171622_1680x1050_scrot](https://user-images.githubusercontent.com/14232936/53184047-ab17b880-35fc-11e9-82b9-2ba99ba3e50d.png)

**Comment by Flamefire** [comment]
Yes the font is not mono-space. Feel free to open a PR fixing the wrong line breaks :)

**Comment by stefson** [comment]
I started by fixing some typos and errors, I'll try to push a fix when it got merged.
